src/core/org/apache/jmeter/report/processor/graph/impl/LatencyOverTimeGraphConsumer.java

```diff
@@ -50,6 +52,39 @@ protected TimeStampKeysSelector createTimeStampKeysSelector() {
         return keysSelector;
     }
 
+    /**
+     * Creates the group info for elapsed time percentile depending on jmeter
+     * properties.
+     *
+     * @param propertyKey
+     *            the property key
+     * @param defaultValue
+     *            the default value
+     * @param serieName Serie name
+     * @return the group info
+     */
+    private GroupInfo createPercentileGroupInfo(String propertyKey, int defaultValue, String serieName) {
```

Comment:
This method duplicated ResponseTimePercentilesOverTimeGraphConsumer#createPercentileGroupInfo. 
Could you try to factor out code ?

Reply:
Move it to the parent class `AbstractOverTimeGraphConsumer.java`? Shouldn't be an issue